.travis.yml

```diff
@@ -28,9 +28,15 @@ env:
     - DD_CASHER_DIR=/tmp/casher
   matrix:
     - TRAVIS_FLAVOR=default
-    - TRAVIS_FLAVOR=core_integration
     - TRAVIS_FLAVOR=checks_mock
 
+matrix:
+  include:
```

Comment:
This only has an impact of about 40 seconds, and now that we've streamlined the CI after the SDK split, that shouldn't be a problem 👍